types/fbdl-core/index.d.ts

```diff
@@ -0,0 +1,80 @@
+// Type definitions for fbdl-core 1.0
+// Project: https://github.com/Snowflake107/fbdl-core#readme
+// Definitions by: 1chiSensei <https://github.com/1chiSensei>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { Readable } from 'stream';
+
+interface Response {
+    name: string;
+    title: string;
+    description: string;
+    rawVideo: string;
+    thumbnail: string;
+    uploadedAt: Date;
+    duration: string;
+    interactionCount: number;
+    streamURL: string;
+    publishedAt: Date;
+    width: number;
+    height: number;
+    nsfw: boolean;
+    genre: string;
+    keywords: any[];
+    comments: number;
+    size: string;
+    quality: string;
+    author: {
+        type: string;
+        name: string;
+        url: string;
+    };
+    publisher: {
+        type: string;
+        name: string;
+        url: string;
+        avatar: string;
+    };
+    url: string;
+    reactions: {
+        total: number;
+        like: number;
+        love: number;
+        care: number;
+        wow: number;
+        haha: number;
+        sad: number;
+        angry: number;
+    };
+    shares: string;
+    views: string;
+}
+
+declare class FBDL {
+    /**
+     * Validates facebook url
+     */
+    static validateURL(url: string): boolean;
+
+    /**
+     * Downloads facebook video
+     */
+    static download(url: string): Promise<Readable>;
+
+    /**
+     * Fetches facebook video info
+     */
+    static getInfo(url: string): Promise<Response | null>;
+
+    /**
+     * Parses time in ms
+     */
+    static parseTime(duration: string): string;
+
+    /** @ignore */
+    private _parseHTML(url: string): Promise<string>;
```

Comment:
This is expected to be private, any reason to incorporate this in the package (which is not extensible, just exposes static methods)

Reply:
When I use a namespace to expose the functions dtslint gives me the error that I should use `export function something` but the ci Failed when i used `export function` and that I should use `export =`